modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -564,6 +564,30 @@ def copy(self):
         """
         pass
 
+    def nested_expressions(
+        self,
+    ) -> Generator[Type["BaseExpr"], Type["BaseExpr"], Type["BaseExpr"]]:
```

Comment:
"If your generator will only yield values, set the SendType and **ReturnType** to `None`:" https://docs.python.org/3/library/typing.html#typing.Generator

```suggestion
    ) -> Generator[Type["BaseExpr"], Type["BaseExpr"], None]:
```